packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+import * as React from "react";
+import {css, StyleSheet} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+
+// TODO(scottgrant): Make sure the toolbars are responsive.
+// import type {MediaSize} from "@khanacademy/wonder-blocks-core";
+
+const dismissIcon =
+    "M12.003 10.588L7.707 6.293a1 1 0 0 0-1.414 1.414l4.295 4.296-4.295 4.295a1 1 0 0 0 1.414 1.414l4.296-4.295 4.295 4.295a1 1 0 0 0 1.414-1.414l-4.295-4.295 4.295-4.296a1 1 0 1 0-1.414-1.414l-4.295 4.295z";
+
+type Props = {
+    /**
+     * A list of nodes to render on the left side of the toolbar. In most
+     * cases, this will be empty.
+     */
+    leftContent?: Array<React.Node>,
+
+    /**
+     * The function to call when dismissing the toolbar. Only relevant if
+     * showClose is true.
+     */
+    onClose?: () => void,
+
+    /**
+     * A list of nodes to render on the right side of the toolbar. This will
+     * typically include buttons, links, span elements with text, or nothing
+     * at all.
+     */
+    rightContent?: Array<React.Node>,
+
+    /**
+     * True, if we should show the dismiss icon on the left side o the toolbar.
+     */
+    showClose: boolean,
+
+    /**
+     * How much vertical space to use for the toolbar. If this prop is not
+     * provided, the default is "medium".
+     */
+    size?: "small" | "medium",
+
+    /**
+     * Additional styling for the toolbar. If a color value is provided and
+     * the showClose prop is set to true, the close icon will use this color.
```

Comment:
This seems overly permissive.  We should probably go with `color: "dark" | "light"` to follow what was done in the modals.  According to design the only two colors allowed are `white` and `darkBlue`.  Are there are any other styles besides color that we need to allow?

Reply:
Thank you, good call. I've had a bit of back-and-forth over how general or specific we want these components to be. You're right though; since we only have the two styles, it makes sense to stick to those for now. I've updated the component here (and sadly stuck with the standardized spelling of `color` with no u)